projects/jdwp/defs/schema.py

```diff
@@ -32,30 +31,63 @@ class PrimitiveType(Enum):
     LOCATION = "location"
 
 
-class Array(Enum):
+class IntegralType(Enum):
+    """Integral type class."""
+
+    INT = "int"
+    BYTE = "byte"
+
+
+@dataclass(frozen=True)
+class ArrayLength:
+    """Array length class."""
+
+    type: IntegralType
+
+
+@dataclass(frozen=True)
+class Array:
     """Array class type."""
 
-    pass
+    element_type: Struct | TaggedUnion
```

Comment:
Why do we need `TaggedUnion` here ?

Reply:
I worked on that, initially it was for the Array to accept the type `TaggedUnion`